tests/snapshots/tests.integration.test_integration.test_sampling.snap

```diff
@@ -0,0 +1,158 @@
+[[{"name" "trace1"
+   "service" nil
+   "resource" "trace1"
+   "error" 0
+   "span_id" 0
+   "trace_id" 0
+   "parent_id" nil
+   "start" 1611739614070324000
+   "duration" 69000
+   "meta" {"runtime-id" "88c29c5e9e1445a8bcce78bd1238a4d4"}
+   "metrics" {"_dd.agent_psr" 1.0
+              "_sampling_priority_v1" 1
+              "system.pid" 10144}}
+      {"name" "child"
+       "service" nil
+       "resource" "child"
+       "error" 0
+       "span_id" 1
+       "trace_id" 0
+       "parent_id" 0
+       "start" 1611739614070374000
+       "duration" 10000}]
+ [{"name" "trace2"
+   "service" nil
+   "resource" "trace2"
+   "error" 0
+   "span_id" 0
+   "trace_id" 1
+   "parent_id" nil
+   "start" 1611739614070713000
+   "duration" 63000
+   "meta" {"runtime-id" "88c29c5e9e1445a8bcce78bd1238a4d4"}
+   "metrics" {"_dd.limit_psr" 1.0
```

Comment:
@brettlangdon does this tag have to be on every root span? I don't see it on every root span.

Reply:
which root spans are you talking about? trace root, chunk root, service root?

Reply:
trace root